pages/instructorAssessmentInstances/instructorAssessmentInstancesClientJS.ejs

```diff
@@ -3,6 +3,20 @@ var bsTable;
 $(function() {
 
     bsTable = $('#usersTable').bootstrapTable({
+        buttons: {
+            studentsOnly: {
+                text: 'Students Only',
+                icon: 'fa-user-graduate',
+                event: () => {
+                    const table = $('#usersTable');
+                    const filterOn = !table.data('filter-student-only');
+                    table.data('filter-student-only', filterOn);
+
+                    $('.columns button[name=studentsOnly]').toggleClass('active', filterOn);
+                    table.bootstrapTable('filterBy', filterOn ? { role: 'Student' } : {});
```

Comment:
It looks like this button defaults to "off" (that is, by default we show all users). Is that correct? I like this default.

Reply:
Yes, it defaults to show all users.

Reply:
To clarify, do you mean "I like the default to stay the way it is" or "I'd like this filter to become on by default"?

Reply:
I like the default to stay the way it is 😄